src/components/modules/PrevNextButtons/Button.tsx

```diff
@@ -14,15 +14,13 @@ interface Props {
 export default class Button extends PureComponent<Props>{
     render() {
         const { buttonText, onPress, link, buttonWidth, buttonActive, buttonDisabled } = this.props;
-        const isDisabled = link.length ? false : true;
-
-        console.log(buttonText, isDisabled);
+        const isDisabled = link === '' ? false : true;
```

Comment:
oh the ? : part reverses the logic, it should just be:
`const isDisabled = link === '';`

I'll fix it when I merge the patch in